### DISCUSSION TITLE
Ceph stretch mode connect to local datacenter

### DISCUSSION THREAD

**Discussion by perrfect** [ceph/ceph-csi#4412] [Q&A]
Hello.
I have a ceph cluster which works in stretch mode:
**DC1:**
node1 (osd, mon, mgr)
node2 (osd, mon)
node3 (osd, mds)

**DC2:**
node1 (osd, mon, mgr)
node2 (osd, mon)
node3 (osd, mds)

**DC3:**
node1 (tiebreaker mon)

Datacenters are distributions  between different  locations.
I use RBD on my clients as CSI plugin in nomad.
How can I set up my clients to connect to the local datacenter?
I don't need big traffic between datacenters only for replications, so my
client in DC1 should connect to ceph in DC1. But when something happened
with DC1 my clients in DC1 should still be working.
Is this possible at all?

**Comment by nixpanic** [comment]
This is (mostly) possible with topology aware provisioning, but that is only implemented for Kubernetes at the moment.
